src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyle = {};
+  const arrStr = sourceString.split(';');
```

Comment:
Consider renaming `arrStr` to a more descriptive name, such as `styleEntries`, to improve code readability.